client/app/components/projects/planning-project-card.hbs

```diff
@@ -5,6 +5,15 @@
 >
   <h3 class="no-margin">
     {{@project.dcpProjectname}}
+
+    {{#if (eq @project.statuscode "On-Hold")}}
+      <span class="text-red text-weight-normal"
+            data-test-project-on-hold
+      > 
+        (On Hold) 
+      </span>
+    {{/if}}
+
```

Comment:
What is the difference between applicant-project-card and planning-project-card components? I'm confused what these are doing and why're they're separate things. They share a lot of template logic.

Reply:
@allthesignals yea they're basically the same now & was going to refactor, just wanted to get this into QA and then take a step back and make that consolidation.